src/plrmodes/marisa.c

```diff
@@ -6,6 +6,8 @@
  * Copyright (c) 2012-2019, Andrei Alexeyev <akari@taisei-project.org>.
  */
 
+#include "cutscenes/cutscene.h"
+#include "progress.h"
```

Comment:
Ditto, but these shouldn't even be necessary here, since `plrmodes.h` pulls those headers in.